presto-accumulo/src/main/java/io/prestosql/plugin/accumulo/io/AccumuloRecordCursor.java

```diff
@@ -147,7 +145,7 @@ public long getCompletedBytes()
     @Override
     public long getReadTimeNanos()
     {
-        return nanoStart > 0L ? (nanoEnd == 0 ? System.nanoTime() : nanoEnd) - nanoStart : 0L;
```

Comment:
should we fix that instead?

Reply:
we could, but i was lazy.

in fact, large majority of the implementations is like that -- they return 0.

Reply:
Why is this broken? While confusing, it seems to work correctly

Reply:
I don't think this can be implemented for this connector.  You really need support from the underlying client to know which calls perform reads.